docs/modules/ROOT/pages/explanation/decisions/dns-naming.adoc

```diff
@@ -0,0 +1,42 @@
+= DNS naming scheme
+
+== Problem
+
+{product} is a web based service offering.
+As such it is accessible through several domains.
+We need a common understanding how we name things and what domains to use for making it accessible.
+
+== Alternatives
+
+In the past, we used different domains for applications managed by us and applications deployed by users, due to a concern regarding cross-domain cookie access. 
+
+== Decision
+
+The base domain will be `appuio.cloud`.
+On the third level, we will serve services that are zone independent.
+Services bound to a specific zone, will be served through the fourth level where the third level will be the zone's identifier.
+Apps served through a zone's default domain use the subdomain `apps` of a zone's base domain.
+
+.List of examples
+====
+* `appuio.cloud` and `www.appuio.cloud` are an exception to the above rule and will be used to host the marketing homepage for {product}.
+* `id.appuio.cloud` The identity provider for {product}.
+* `console.appuio.cloud` A web application to manage cross zone concerns.
+* `console.<zone>.appuio.cloud` The OpenShift console of a zone.
+* `<route>.apps.<zone>.appuio.cloud` A route hosted on a zone relying on the zones default app domain.
+====
+
+The content of `<route>` is covered by an OpenShift core feature and will consist of the name of the route object and the namespace it is located in.
+What goes into `<zone>` is covered by xref:appuio-cloud:ROOT:explanation/decisions/zone-naming.adoc[Zone naming scheme].
```

Comment:
Similar to my other comment, the Alternatives section should contain complete proposals, whereas the Decision section only referring to which Alternative choosen.

So I'd move the content of the Decision section to an Alternative. And then add an example for a sentence of "In the past, we used different domains for applications managed by us and applications deployed by users, due to a concern regarding cross-domain cookie access." -- I don't quite understand this. How did that look like?

Reply:
See other comment